### ISSUE TITLE
`write_xlsx()` can not write `data.table`

### ISSUE THREAD

**Issue by djhurio** [JanMarvin/openxlsx2#312]
It is not possible to write a `data.table::data.table` with `write_xlsx()`. It is because of conflicting `rbindlist()` functions from `data.table` and `openxlsx2` packages. It would be handy to allow writing `data.table` objects directly with `write_xlsx()`. BTW, it is possible with the `openxlsx::write.xlsx()`. Thanks!

Code to reproduce the error:

```
library(data.table)
library(openxlsx2)
write_xlsx(x = data.frame(1), file = "temp.xlsx")
write_xlsx(x = data.table(1), file = "temp.xlsx")
```

Result:

```
> write_xlsx(x = data.table(1), file = "temp.xlsx")
Error in rbindlist(l, use.names, fill, idcol) : 
  Column 1 ['V1'] of item 2 is missing in item 1. Use fill=TRUE to fill with NA (NULL for list columns), or use.names=FALSE to ignore column names.
```

**Comment by JanMarvin** [comment]
Hi @djhurio , thanks for the report. There is no need for us to export rbindlist, it's an internal function and not required to be exported.

**Comment by JanMarvin** [comment]
A fix is in #313. It's been a long day, I'll check it again tomorrow morning, but most likely it will be the next one to merge.

```R
> library(data.table)
> library(openxlsx2)
> 
> write_xlsx(x = data.frame(1), file = "temp.xlsx")
> read_xlsx("temp.xlsx")
  X1
2  1
> write_xlsx(x = data.table(1), file = "temp.xlsx")
> read_xlsx("temp.xlsx")
  V1
2  1
```
